cli/commands/benchmark-web-vitals.mjs

```diff
@@ -164,13 +164,28 @@ async function benchmarkURL( browser, params ) {
 			);
 
 		// Load the page.
-		const response = await page.goto(
-			`${ params.url }?rnd=${ requestNum }`,
-			{ waitUntil: 'networkidle0' }
-		);
+		const url = new URL( params.url );
+		url.searchParams.append( 'rnd', String( requestNum ) );
```

Comment:
Question: Rather than using `requestNum` here, shouldn't an actual random number be used or a UUID? Otherwise, when repeated calls are made there will be duplicate requests to `/?rnd=0` which could be cached and would produce misleading results.

Reply:
Good point, I think that makes sense.

Reply:
How about just this:

```suggestion
		url.searchParams.append( 'rnd', String( Math.random() ) );
```